tests/bluetooth/tester/src/btp/btp_mcp.h

```diff
@@ -0,0 +1,290 @@
+	uint8_t id[6];
+} __packed;
+
+#define BTP_MCP_PARENT_GROUP_OBJ_ID_READ	0x0c
+struct btp_mcp_parent_group_obj_id_read_cmd {
+	bt_addr_le_t address;
+} __packed;
+
+#define BTP_MCP_CURRENT_GROUP_OBJ_ID_READ	0x0d
+struct btp_mcp_current_group_obj_id_read_cmd {
+	bt_addr_le_t address;
+} __packed;
+
+#define BTP_MCP_CURRENT_GROUP_OBJ_ID_SET	0x0e
+struct btp_mcp_current_group_obj_id_set_cmd {
+	bt_addr_le_t address;
+	uint8_t id[6];
+} __packed;
+
+#define BTP_MCP_PLAYING_ORDER_READ		0x0f
+struct btp_mcp_playing_order_read_cmd {
+	bt_addr_le_t address;
+} __packed;
+
+#define BTP_MCP_PLAYING_ORDER_SET		0x10
+struct btp_mcp_playing_order_set_cmd {
+	bt_addr_le_t address;
+	uint8_t order;
+} __packed;
+
+#define BTP_MCP_PLAYING_ORDERS_SUPPORTED_READ	0x11
+struct btp_mcp_playing_orders_supported_read_cmd {
+	bt_addr_le_t address;
+} __packed;
+
+#define BTP_MCP_MEDIA_STATE_READ		0x12
+struct btp_mcp_media_state_read_cmd {
+	bt_addr_le_t address;
+} __packed;
+
+#define BTP_MCP_OPCODES_SUPPORTED_READ		0x13
+struct btp_mcp_opcodes_supported_read_cmd {
+	bt_addr_le_t address;
+} __packed;
+
+#define BTP_MCP_CONTENT_CONTROL_ID_READ		0x14
+struct btp_mcp_content_control_id_read_cmd {
+	bt_addr_le_t address;
+} __packed;
+
+#define BTP_MCP_SEGMENTS_OBJ_ID_READ		0x15
+struct btp_mcp_segments_obj_id_read_cmd {
+	bt_addr_le_t address;
+} __packed;
+
+#define BTP_MCP_CURRENT_TRACK_OBJ_ID_READ	0x16
+struct btp_mcp_current_track_obj_id_read_cmd {
+	bt_addr_le_t address;
+} __packed;
+
+#define BTP_MCP_CURRENT_TRACK_OBJ_ID_SET	0x17
+struct btp_mcp_current_track_obj_id_set_cmd {
+	bt_addr_le_t address;
+	uint8_t id[6];
+} __packed;
+
+#define BTP_MCP_CMD_SEND			0x18
+struct btp_mcp_send_cmd {
+	bt_addr_le_t address;
+	uint8_t  opcode;
+	uint8_t use_param;
+	int32_t param;
+} __packed;
+
+#define BTP_MCP_CMD_SEARCH			0x19
+struct btp_mcp_search_cmd {
+	bt_addr_le_t address;
+	uint8_t type;
+	uint8_t param_len;
+	uint8_t param[];
+} __packed;
+
+/* MCP events */
+#define BTP_MCP_DISCOVERED_EV			0x80
+struct btp_mcp_discovered_ev {
+	bt_addr_le_t address;
+	uint8_t status;
+} __packed;
+
+#define BTP_MCP_TRACK_DURATION_EV		0x81
+struct btp_mcp_track_duration_ev {
+	bt_addr_le_t address;
+	uint8_t status;
+	int32_t dur;
+} __packed;
+
+#define BTP_MCP_TRACK_POSITION_EV		0x82
+struct btp_mcp_track_position_ev {
+	bt_addr_le_t address;
+	uint8_t status;
+	int32_t pos;
+} __packed;
+
+#define BTP_MCP_PLAYBACK_SPEED_EV		0x83
+struct btp_mcp_playback_speed_ev {
+	bt_addr_le_t address;
+	uint8_t status;
+	int8_t speed;
+} __packed;
+
+#define BTP_MCP_SEEKING_SPEED_EV		0x84
+struct btp_mcp_seeking_speed_ev {
+	bt_addr_le_t address;
+	uint8_t status;
+	int8_t speed;
+} __packed;
+
+#define BTP_MCP_ICON_OBJ_ID_EV			0x85
+struct btp_mcp_icon_obj_id_ev {
+	bt_addr_le_t address;
+	uint8_t status;
+	uint8_t id[6];
+} __packed;
+
+#define BTP_MCP_NEXT_TRACK_OBJ_ID_EV		0x86
+struct btp_mcp_next_track_obj_id_ev {
+	bt_addr_le_t address;
+	uint8_t status;
+	uint8_t id[6];
```

Comment:
Consider using 
```suggestion
	uint8_t id[BT_OTS_OBJ_ID_SIZE];
```

For these

Reply:
Changed